web/api/v1/api.go

```diff
@@ -436,6 +440,17 @@ func (api *API) alertmanagers(r *http.Request) (interface{}, *apiError) {
 	return ams, nil
 }
 
+type prometheusConfig struct {
+	RawFormat string `json:"rawFormat"`
```

Comment:
agreed, as this is parsed and marshaled again to yaml, it makes most sense to call this maybe `yaml` instead